ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -364,7 +352,7 @@ public struct FluentNotification: View, TokenizedControlView {
     }
 
     private func presentAnimated() {
-        withAnimation(.spring(response: state.style.animationDurationForShow,
+        withAnimation(.spring(response: state.style.animationDurationForShow / 2.0,
```

Comment:
is this a difference between uikit's `withDuration` parameter and swift's `response`? because the uikit one still remains animationDurationForShow, not animationDurationForShow/2

Reply:
I'll be honest, I have no idea, but they are definitely off by exactly a factor of 2 (I checked with a stopwatch).

Reply:
ENGINEERING